pkg/controllers/provisioning/batcher.go

```diff
@@ -22,7 +22,7 @@ import (
 
 var (
 	MaxBatchDuration  = time.Second * 10
-	BatchIdleDuration = time.Second * 1
+	BatchIdleDuration = time.Second * 8
```

Comment:
What should the batch duration be? Currently 1 second is too short for grouping pods.

Reply:
set it back to one. We can revisit this later. Maybe allow the user to adjust it will provide some flexibility for different use cases.